### PR TITLE
showing skills of members

### PR DESCRIPTION
**Pull request by akshay1502**
Screenshot (5)
Screenshot (4)
I have created a skills.js file that has data that includes the technology name and background color for it.

### CODE REVIEW

File: components/member-card/card.module.scss [hunk 1]
```diff
@@ -33,8 +34,10 @@
 }
 
 .imgContainer {
-  width: 180px;
-  height: 180px;
+  // width: 180px;   og
+  // height: 180px;
```
Comment (sumitd94) [inline components/member-card/card.module.scss:38]: Please remove this if not needed

File: components/member-card/card.module.scss [hunk 2]
```diff
@@ -33,8 +34,10 @@
...
+  width: 150px;
+  height: 150px;
```
Comment (sumitd94) [inline components/member-card/card.module.scss:40]: can we please use `rem`? I know there are places where we are using `px`, but I am making sure any new changes we have uses `rem`. There will be a refactor coming soon where the `px` will be changed to `rem`

File: components/member-card/card.module.scss [hunk 3]
```diff
@@ -63,3 +66,18 @@
...
+
+.showskills{
+  display: flex;
+  flex-wrap: wrap;
+  margin:10px 0;
+  justify-content: center;
+}
+
+.techskills{
+  margin: 2px 5px;
```
Comment (sumitd94) [inline components/member-card/card.module.scss:78]: rem?

File: components/member-card/card.module.scss [hunk 4]
```diff
@@ -63,3 +66,18 @@
...
+  color: #ffffff;
+  padding: 3px 5px;
```
Comment (sumitd94) [inline components/member-card/card.module.scss:80]: rem?

File: components/member-card/index.js [hunk 5]
```diff
@@ -3,6 +3,8 @@ import classNames from './card.module.scss';
 import { motion } from 'framer-motion';
 import SocialMediaIcon from '../social-media-icon';
 import PropTypes from 'prop-types';
+// import { skills } from './skills';
```
Comment (sumitd94) [inline components/member-card/index.js:6]: please remove this if not needed

File: components/member-card/showskills.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import { skills } from './skills';
+import classNames from './card.module.scss';
+
+const Showskills = () => {
+  //   skills.map(skill => console.log(skill));
```
Comment (sumitd94) [inline components/member-card/showskills.js:5]: please remove this if not needed

File: components/member-card/showskills.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+  return (
+    <div className={classNames.showskills}>
+      {skills.map((skill) => {
+        // eslint-disable-next-line react/jsx-key
```
Comment (sumitd94) [inline components/member-card/showskills.js:9]: please remove this and fix the lint errors

File: components/member-card/showskills.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+        return (
+          // eslint-disable-next-line react/jsx-key
```
Comment (sumitd94) [inline components/member-card/showskills.js:11]: RE:

File: components/member-card/showskills.js [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+          <p style={{ background: `${skill.color}` }} className={classNames.techskills}>
```
Comment (sumitd94) [inline components/member-card/showskills.js:12]: `key` attribute missing

File: components/member-card/skills.js [hunk 10]
```diff
@@ -0,0 +1,29 @@
+// import React from "react";
+
+// export const skills = ['NodeJs', 'ReactJs', 'MongoDB', 'EmberJs', 'Javascript', 'Firestore'];
```
Comment (sumitd94) [inline components/member-card/skills.js:3]: please remove this

**Review by sumitd94 — CHANGES_REQUESTED** [review]
Few changes requested, please

**Comment by akshay1502** [comment]
Screenshot (6)
updated css for showing skills

**Comment by akshay1502** [comment]
Screenshot (7)

File: components/member-card/showskills.js [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+          </p>
+        );
+      })}
+      {show &&
+        skills_data.skills.map((skill) => {
+          return (
+            <p
+              style={{ background: `${skill.background}` }}
+              className={classNames.techskills}
+              key={skill.color}>
```
Comment (sumitd94) [inline components/member-card/showskills.js:24]: why do you think using `color` as the key is a good practice? what if any two skills have the same color? can we please change to use a unique id?

File: components/member-card/showskills.js [hunk 12]
```diff
@@ -0,0 +1,36 @@
+import { skills_data } from './skills';
+import classNames from './card.module.scss';
+import PropTypes from 'prop-types';
+
+const Showskills = ({ show }) => {
+  return (
+    <div className={classNames.showskills}>
```
Comment (sumitd94) [inline components/member-card/showskills.js:7]: please use the classNames to be `camelCase`

so - `showSkills`

File: components/member-card/showskills.js [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+      {skills_data.topskills.map((skill) => {
+        return (
+          <p
+            style={{ background: `${skill.background}` }}
+            className={classNames.techskills}
```
Comment (sumitd94) [inline components/member-card/showskills.js:12]: camelCase

File: components/member-card/showskills.js [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+            {skill.tech}
+          </p>
+        );
+      })}
+      {show &&
+        skills_data.skills.map((skill) => {
+          return (
+            <p
+              style={{ background: `${skill.background}` }}
+              className={classNames.techskills}
```
Comment (sumitd94) [inline components/member-card/showskills.js:23]: camelCase

File: components/member-card/skills.js [hunk 15]
```diff
@@ -0,0 +1,30 @@
...
+    {
+      tech: 'Javascript',
+      background: '#D9C643'
+    },
+    {
+      tech: 'Firestore',
+      background: '#FFA611'
+    }
+  ]
+};
```
Comment (sumitd94) [inline components/member-card/skills.js:30]: can we please move this into a Mock Data folder? create a new folder in the root, `mock-data`, and move this file there.

File: components/member-profile/index.js [hunk 16]
```diff
@@ -6,6 +6,7 @@ import classNames from './member-profile.module.scss';
 import ContributionType from './contribution-type/';
 import { motion } from 'framer-motion';
 import Modal from 'components/modal';
+import Showskills from '../member-card/showskills';
```
Comment (sumitd94) [inline components/member-profile/index.js:9]: please make use of absolute imports.
`components/member-card/show-skills`

File: components/member-card/showskills.js [hunk 17]
```diff
@@ -0,0 +1,36 @@
+import { skills_data } from './skills';
+import classNames from './card.module.scss';
+import PropTypes from 'prop-types';
+
+const Showskills = ({ show }) => {
```
Comment (sumitd94) [inline components/member-card/showskills.js:5]: should be `ShowSkills`

**Review by sumitd94 — CHANGES_REQUESTED** [review]
Few changes requested, please. Also, can you make sure the name of the component file is `show-skills.js`

File: components/member-card/showskills.js (hunk 11 above)
Comment (akshay1502) [reply components/member-card/showskills.js]: added id property.

File: components/member-card/showskills.js (hunk 12 above)
Comment (akshay1502) [reply components/member-card/showskills.js]: done.

File: components/member-card/showskills.js (hunk 13 above)
Comment (akshay1502) [reply components/member-card/showskills.js]: done.

File: components/member-card/showskills.js (hunk 14 above)
Comment (akshay1502) [reply components/member-card/showskills.js]: done. Made class name to camelCase.

File: components/member-profile/index.js (hunk 16 above)
Comment (akshay1502) [reply components/member-profile/index.js]: Now using absolute imports.

**Comment by akshay1502** [comment]
Screenshot (8)
Screenshot (9)
We will need color codes for other tech skill badges too.

File: components/member-card/card.module.scss [hunk 18]
```diff
@@ -63,3 +63,31 @@
...
+  margin:10px 0;
+  justify-content: center;
+}
+
+.showMemberSkills{
+  display: flex;
+  flex-wrap: wrap;
+  margin:10px 0;
+  justify-content: center;
+}
```
Comment (sumitd94) [inline components/member-card/card.module.scss:79]: The same CSS is being repeated, can we just use the same class name at both places?

**Review by sumitd94 — CHANGES_REQUESTED** [review]
one small change, after that we can merge this.

**Review by sumitd94 — APPROVED** [review]

**Pull request merged**